public/app/plugins/datasource/cloud-monitoring/module.ts

```diff
@@ -1,12 +1,14 @@
 import { DataSourcePlugin } from '@grafana/data';
 import CloudMonitoringDatasource from './datasource';
 import { QueryEditor } from './components/QueryEditor';
+import CloudMonitoringCheatSheet from './components/CloudMonitoringCheatSheet';
 import { CloudMonitoringConfigCtrl } from './config_ctrl';
 import { CloudMonitoringAnnotationsQueryCtrl } from './annotations_query_ctrl';
 import { CloudMonitoringVariableQueryEditor } from './components/VariableQueryEditor';
 import { CloudMonitoringQuery } from './types';
 
 export const plugin = new DataSourcePlugin<CloudMonitoringDatasource, CloudMonitoringQuery>(CloudMonitoringDatasource)
+  .setQueryEditorHelp(CloudMonitoringCheatSheet)
```

Comment:
interesting, I didn't know about this!!!

Reply:
yep it's pretty neat!